src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -59,13 +59,13 @@ const TableHeader: React.FC<TableHeaderProps> = ({
       return false;
     }
     // Locked state is in the experience record
-    return experiences?.[venueName]?.tables?.[table]?.locked;
+    return experience?.tables?.[table]?.locked;
```

Comment:
Can you explain this? I'm guessing that at some point the structure of how we were keeping this data changed to no longer have the `venueName` in it like this?

Reply:
The query that fetches and storeAs gets the experience by venue name already. Which means what `experiences` is actually 1 single experience for your current venue.